daras_ai_v2/language_model.py

```diff
@@ -1964,11 +1964,27 @@ def entry_to_prompt_str(entry: ConversationEntry) -> str:
 
 
 def format_chat_entry(
-    *, role: str, content: str, images: list[str] = None
+    *,
+    role: str,
+    content_text: str,
+    input_images: typing.Optional[list[str]] = None,
+    input_audio: typing.Optional[str] = None,
+    input_documents: typing.Optional[list[str]] = None,
+    render_input_urls: typing.Optional[bool] = False,
 ) -> ConversationEntry:
-    if images:
+
+    input_urls = []
+    if input_images and not render_input_urls:
```

Comment:
this condition didn't make a lot of sense. the parameter is called render_input_urls, but the function then goes on to do the exact opposite (i.e. NOT render URLs)? better naming please